staging/src/k8s.io/client-go/pkg/apis/clientauthentication/v1beta1/conversion.go

```diff
@@ -17,10 +17,10 @@ limitations under the License.
 package v1beta1
 
 import (
-	conversion "k8s.io/apimachinery/pkg/conversion"
-	clientauthentication "k8s.io/client-go/pkg/apis/clientauthentication"
+	"k8s.io/apimachinery/pkg/conversion"
+	"k8s.io/client-go/pkg/apis/clientauthentication"
 )
 
 func Convert_clientauthentication_ExecCredentialSpec_To_v1beta1_ExecCredentialSpec(in *clientauthentication.ExecCredentialSpec, out *ExecCredentialSpec, s conversion.Scope) error {
-	return nil
```

Comment:
I'm very confused about what was happening here before :)

Reply:
Spec was empty for the beta type before.

Reply:
I guess I meant why a custom conversion function was even present before

Reply:
See above.